src/Microsoft.Diagnostics.Monitoring.EventPipe/Configuration/GcCollectConfiguration.cs

```diff
@@ -10,7 +10,8 @@ public sealed class GcCollectConfiguration : MonitoringSourceConfiguration
     {
         public GcCollectConfiguration()
         {
-            RequestRundown = false;
+            RundownKeyword = (long)Tracing.Parsers.ClrTraceEventParser.Keywords.GC;
+            RetryStrategy = RetryStrategy.DropKeywordDropRundown;
```

Comment:
How expensive is this new rundown? How does it scale compared to the regular rundown?

Reply:
The GC only rundown will collect only a single event that describe the GC settings (e.g. did we turn on background GC, ...), in particular, we will skip all the work related to types and methods, which should mean this should be much faster than a standard rundown.